SmartDeviceLink/SDLIAPSession.m

```diff
@@ -27,10 +34,18 @@ - (instancetype)initWithAccessory:(EAAccessory *)accessory forProtocol:(NSString
 
     self = [super init];
     if (self) {
+        if ([protocol isEqualToString:@"com.smartdevicelink.prot0"]){
+            _isDataSession = NO;
+        }
+        else{
+            _isDataSession = YES;
+        }
```

Comment:
We could shorten this a bit `_isDataSession = [protocol isEqualToString:@"com.smartdevicelink.prot0"] ? NO : YES;`